development/cordova/README.md

```diff
@@ -0,0 +1,31 @@
+# Development - Cordova
+
+Note: these instructions assume you are building an Angular 1.4 based
+application.
+
+To begin a Cordova 5.0.0 project, start with a `package.json` file. It is best
+practice to install dependencies within a project rather than globally, which is
+why you shouldn't follow the [Cordova instructions](http://cordova.apache.org/docs/en/5.0.0/guide_cli_index.md.html#The%20Command-Line%20Interface)
+to the letter.
+
+```
+$ echo "{}" > package.json
+$ npm install --save angular@1.4.0 angular-resource@1.4.0 angular-route@1.4.0 \
+> cordova@5.0.0 moment@2.10.2
+$ npm install --save-dev angular-mocks@1.4.0 appium@1.4.0 chai@2.1.2 \
+> chai-as-promised@5.0.0 eslint@0.19.0 mocha@2.2.1 sinon@1.14.1 testem@0.8.0-0 \
+> wd@0.3.11
+```
```

Comment:
@ericcf I am not sure the best place for this, but I am going to make mention of one more thing and have @eschlange note it.  After you finish adding cordova and you need to add the android platform - similar to how Eric is adding npm's above - it may be helpful to note which android platform you are adding: `./node_modules/.bin/cordova platform add android@4.0.0`.  In the case here, I added platform 4.0.0.